tools/generate_vsn.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/bash
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+#!/bin/sh
+DIR="$( cd "$( dirname "$0" )" && pwd )"
 GIT_VSN=`git describe --always --tags 2>/dev/null`
```

Comment:
fine.

But both versions assume, that we are running from one of directories of the repo.
Would still work differently for *both* versions if called from repo and from outside of repo.

Reply:
would replace with something like:

```bash
#!/bin/sh

# Go to repo root
cd `dirname "$0"`/../

GIT_VSN=`git describe --always --tags 2>/dev/null`
if [ $? -eq 0 ]; then
    echo $GIT_VSN
else
    cat VERSION
fi
```

Reply:
@arcusfelis thanks for the tip. I applied it and merge the PR into master.